go/vt/vtorc/discovery/queue.go

```diff
@@ -30,39 +30,41 @@ import (
 	"time"
 
 	"vitess.io/vitess/go/vt/log"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/topo/topoproto"
 	"vitess.io/vitess/go/vt/vtorc/config"
 )
 
 // queueItem represents an item in the discovery.Queue.
 type queueItem struct {
-	Key      string
-	PushedAt time.Time
+	TabletAlias *topodatapb.TabletAlias
```

Comment:
Interestingly, this reduces the size of the `enqueued` map keys by 50%:

```
package main

import (
        "fmt"
        "unsafe"

        "vitess.io/vitess/go/vt/topo/topoproto"
        topodatapb "vitess.io/vitess/go/vt/proto/topodata"
)

func main() {
        tabletAlias := &topodatapb.TabletAlias{Cell: "zone1", Uid: 12345}
        fmt.Println(unsafe.Sizeof(tabletAlias))

        tabletAliasString := topoproto.TabletAliasString(tabletAlias)
        fmt.Println(unsafe.Sizeof(tabletAliasString))
}
```
Returns:
```bash
8
16
```